.checkpatch.conf

```diff
@@ -0,0 +1,10 @@
+--quiet
+--no-tree
+--strict
+--ignore=FILE_PATH_CHANGES
+--ignore=GERRIT_CHANGE_ID
+--ignore=NOT_UNIFIED_DIFF
+--ignore=CAMELCASE
+--ignore=PREFER_KERNEL_TYPES
+--ignore=CONCATENATED_STRING
+--ignore=USLEEP_RANGE
```

Comment:
Minor: since this is a new file, let's take this opportunity to order the lines alphabetically (`ggvG:sort` for the `vi` afficionados).